onelogin_aws_cli/credentials.py

```diff
@@ -3,12 +3,113 @@
 manner
 """
 import getpass
+from typing import List
 
 import keyring
+from onelogin.api.models.device import Device
 
 from onelogin_aws_cli.configuration import Section
 
 
+class MFACredentials(object):
+    """
+    Class to encapsulate the handling and storage of MFA devices, and
+    retrieving of OTP's
+    """
+
+    def __init__(self):
+        self._interactive = True
+
+        self.reset()
+
+    @property
+    def has_device(self) -> bool:
+        """
+        True is the MFA has an MFA device selected waiting to be used
+        """
+        return (self._device_index is not None) and \
```

Comment:
As a followup I can try to add a config option to preselect an auth device.

Reply:
It's been on my mind to allow an OTP device to be selected.

Created issue in https://github.com/physera/onelogin-aws-cli/issues/62